src/block/tile/TileFactory.php

```diff
@@ -59,6 +59,7 @@ public function __construct(){
 		$this->register(BrewingStand::class, ["BrewingStand", "minecraft:brewing_stand"]);
 		$this->register(Cauldron::class, ["Cauldron", "minecraft:cauldron"]);
 		$this->register(Chest::class, ["Chest", "minecraft:chest"]);
+		$this->register(ChiseledBookshelf::class, ["Chiseled Bookshelf", "minecraft:chiseled_bookshelf"]);
```

Comment:
```suggestion
		$this->register(ChiseledBookshelf::class, ["ChiseledBookshelf", "minecraft:chiseled_bookshelf"]);
```
Vanilla ID does not contain space